lib/scrabble.rb

```diff
@@ -1,5 +1,8 @@
+require "pry"
 class Scrabble
-  def point_values
+  attr_reader :point_values, :scored_letters
```

Comment:
You do not need any attr_readers - the only methods that we need to be able to call are `score` and `score_with_multipliers`